internal/pkg/cli/flag.go

```diff
@@ -34,13 +34,19 @@ const (
 	resourceTagsFlag   = "resource-tags"
 	detachFlag         = "detach"
 
+	// Deploy flags.
+	yesInitWorkloadFlag = "init-wkld"
+	noInitWorkloadFlag  = "no-init-wkld"
```

Comment:
Can we remove `noInitWorkloadFlag` after making the change to consolidate these two flags?